### ISSUE TITLE
Start tag expected, '&lt;' not found don't know what to do

### ISSUE THREAD

**Issue by leifadev** [factmaven/xml-to-json#3]
When I try to use [this](https://api.bestbuy.com/v1/products(sku%20in(6429442))?apiKey=m16fBGHO0ml7HaoS2zlneHcA) url as xml input it will give me `"Start tag expected, '&lt;' not found"`.
I am not too familiar with XML and some of this HTML syntax but it looks like there's a parsing error. 

I believe &lt;, a lesser than sign is being confused with an actual < or something. Can I fix this?

**Comment by ethanosullivan** [comment]
Hi @leifadev, thanks for bringing this to our intention with your details. I replicated the steps you mentioned and I see the same issue. Let me spend some time on this and figure out what is wrong.

**Comment by leifadev** [comment]
Yeah it's all good, I was dumb and should have read the API so I could use it with json. Thanks for looking into it, I will use this in the future!

**Comment by ethanosullivan** [comment]
Glad you solved it, I'll go ahead and close this issue then.

**Comment by alexandre-root** [comment]
I have the same issue, link [here](https://api.factmaven.com/xml-to-json/?xml=https://api.flightplandatabase.com/plan/4387649)
